tests/unit_tests/analysis/test_es_update.py

```diff
@@ -233,31 +233,31 @@ def test_update_report_with_different_observation_status_from_smoother_update(
         (
             "IES_ENKF",
             [
-                0.49758108622091335,
-                -0.9707583934658779,
-                -0.6519965469839432,
-                -0.11707580894540548,
-                -0.019608044356680288,
-                0.05109583740829979,
-                -1.5443581696894637,
-                0.2589327792111955,
-                -0.6606765119887932,
-                0.686502952382529,
+                0.6037999677949758,
```

Comment:
It is different from that before combine (using commit `0a55f0a19 Let monitor emit optional heartbeats` as ref). And yep think so, because it runs IES, which does go into `_load_observations_and_responses`, so it gets out a `S` with a different ordering (not only by name, but by name->index), the primary order by name was there, but secondary ordering by index was arbitrary before AFAIK.

```
        (
            "IES_ENKF",
            [
                0.5167529669896218,
                -0.9178847938402281,
                -0.6299046429604261,
                -0.1632005925319205,
                0.0216488942750398,
                0.07464619425897459,
                -1.5587692532545538,
                0.22910522740018124,
                -0.7171489000139469,
                0.7287252249699406,
            ],
        ),
        (
            "STD_ENKF",
            [
                1.3040645145742686,
                -0.8162878122658299,
                -1.5484856041224397,
                -1.379896334985399,
                -0.510970027650022,
                0.5638868158813687,
                -2.7669280724377487,
                1.7160680670028017,
                -1.2603717378211836,
                1.2014197463741136,
            ],
        ),

```